### PR TITLE
Update some dependencies manually, removing duplicate deps

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
Removes all duplicate deps (as seen by `cargo +stable tree -d`).  Primarily by updating `wasm-debug` but also by through updating some other dependencies.

### CODE REVIEW

**Comment by MarkMcCaskey** [comment]
bors try

File: Cargo.lock [hunk 1]
```diff
@@ -1802,20 +1810,74 @@ version = "0.9.0+wasi-snapshot-preview1"
...
 
+[[package]]
+name = "wasm-bindgen"
+version = "0.2.59"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3557c397ab5a8e347d434782bcd31fc1483d927a6826804cec05cc792ee2519d"
+dependencies = [
+ "cfg-if",
+ "wasm-bindgen-macro",
+]
```
Comment (syrusakbary) [inline Cargo.lock]: Again wasm-bindgen here??

Comment (MarkMcCaskey) [reply Cargo.lock]: This is from criterion via plotters via js-sys, I believe it should be a dev-dependency

Comment (MarkMcCaskey) [reply Cargo.lock]: Which, from their Cargo.toml, seems to be only for `[target.'cfg(target_arch = "wasm32")'.dependencies]`

**Comment by MarkMcCaskey** [comment]
bors try-

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors merge

**Pull request merged**